web/app.js

```diff
@@ -598,6 +601,11 @@ const PDFViewerApplication = {
       eventBus
     );
 
+    this.annotationEditorParams = new AnnotationEditorParams(
+      appConfig.annotationEditorParams,
+      eventBus
+    );
+
```

Comment:
Please place this code in the following if-block instead, since it shouldn't be unconditionally required as far as I can tell.
https://github.com/mozilla/pdf.js/blob/4e025e1f084e5d47794d0be732d4c451783c71b7/web/app.js#L570-L577